packages/buckram/assets/styles/components/specials/_contributors.scss

```diff
@@ -6,63 +6,55 @@
 //                           Contributors                                     //
 //============================================================================//
 
-div.contributors {
+.contributors {
   border-top: 1px solid;
-  margin-top: 2rem;
+  margin-top: $contributors-margin-top;
+  font-family: $body-font-family;
 
-  &.page {
-    border-top: 0;
-
-    h3 {
-      font-family: $font-1;
-      font-size: $h1-font-size;
-    }
-
-    .contributor_name {
-      font-size: 25px;
-    }
-  }
-
-  .contributor_name_and_links {
-    margin-top: 2rem;
+ .about-authors {
+    margin-top: $contributors-margin-top;
+    margin-bottom: $contributors-margin-bottom;
+    text-align: left;
   }
 
-  .about-authors {
-   margin: 2rem 0;
-   text-align: left;
+  // TODO: replace clearfix hack with better CSS Grid solution when Prince supports grid layout
```

Comment:
Clearfix hack so that the div which contains profile picture, name & links will take on the expected height. Prince doesn't support grid layouts yet, so probably best to use this approach?